code/02_create-figures.R

```diff
@@ -0,0 +1,245 @@
+       width = 17.49, height = 9.03, 
+       units = "cm", device = "png", dpi = 600)
+
+
+### 4 - Figure 2 - Health Board Map ----
+
+
+### 5 - Figure 3 - Age/Sex Bar Chart ----
+
+fig3 <- 
+  
+  summarise_data(basefile, age_grp, sex, trend = FALSE) %>%
+  filter(!is.na(sex)) %>%
+  
+  ggplot(aes(x = age_grp, y = qom, fill = sex)) +
+  geom_bar(position = "dodge", stat = "identity", width = 0.5, show.legend = T) +
+  theme(panel.background = element_blank(),
+        panel.grid.major.x = element_blank(),
+        panel.grid.major.y = element_blank(),
+        axis.title.x = element_text(size = 12, face = "bold"),
+        axis.title.y = element_text(size = 12, face = "bold"),
+        axis.text = element_text(size = 12),
+        legend.title = element_blank()) +
+  xlab("Age Group") + 
+  ylab("Percentage")
+
+ggsave(here("markdown", "figures", "figure-3.png"), 
+       plot = fig3,
+       width = 17.49, height = 9.03, 
+       units = "cm", device = "png", dpi = 600)
+
+
+### 6 - Figure 4 ----
+
+fig4 <- 
+  
+  summarise_data(basefile, simd, trend = FALSE) %>%
+  
+  ggplot(aes(x = simd, y = qom, fill = 1)) +
+  geom_bar(position = "dodge", stat = "identity", width = 0.5, show.legend = F) +
+  theme(panel.background = element_blank(),
+        panel.grid.major.x = element_blank(),
+        panel.grid.major.y = element_blank(),
+        axis.title.x = element_text(size = 12, face = "bold"),
+        axis.title.y = element_text(size = 12, face = "bold"),
+        axis.text = element_text(size = 12),
+        legend.title = element_blank()) +
+  xlab("Deprivation") + 
+  ylab("Percentage")
+
+ggsave(here("markdown", "figures", "figure-4.png"), 
+       plot = fig4,
+       width = 17.49, height = 9.03, 
+       units = "cm", device = "png", dpi = 600)
+
+
+### 7 - Figure 5 ----
+
+fig5 <- 
+  
+  summarise_data(basefile, urban_rural, trend = FALSE) %>%
+  
+  ggplot(aes(x = urban_rural, y = qom, fill = 1)) +
+  geom_bar(position = "dodge", stat = "identity", width = 0.5, show.legend = F) +
+  theme(panel.background = element_blank(),
+        panel.grid.major.x = element_blank(),
+        panel.grid.major.y = element_blank(),
+        axis.title.x = element_text(size = 12, face = "bold"),
+        axis.title.y = element_text(size = 12, face = "bold"),
+        axis.text = element_text(size = 12),
+        legend.title = element_blank()) +
+  xlab("Urban / Rural Classification") + 
+  ylab("Percentage")
+
+ggsave(here("markdown", "figures", "figure-5.png"), 
+       plot = fig5,
+       width = 17.49, height = 9.03, 
+       units = "cm", device = "png", dpi = 600)
+
+
+### 8 - Figure A1.1 - Health Board Trends ----
+
+figa11 <- 
+  
+  summarise_data(basefile, hb, trend = TRUE) %>%
+  
+  # Add row for Scotland
+  bind_rows(summarise_data(basefile, trend = TRUE) %>%
+              mutate(hb = "Scotland")) %>%
+  mutate(hb = forcats::fct_relevel(hb, "Scotland")) %>%
+  
+  ggplot(aes(x = fy, y = qom, group = 1)) +
+  geom_line() +
```

Comment:
Can also set up a scale limit for y axis as I noticed they are all from 75 to 100. Just add:
ylim(75, 100) + 

Can do this for all Figure A charts.

Reply:
Is there a reason we need to do this? It's important that all the faceted charts have the same scale but I think that is guaranteed with the code as is? Adding the ylim line would risk data being cut off if it happened to be less than 75 would it not?

Reply:
When I ran the code as it is, the scale is automatically set based on the data. Does it matter if one batch of charts start from 75, the other batch start from 80? If it doesn't matter, then that's fine not using ylim. I just noticed the faceted charts in the last publication report all start from 75. Definitely we will have the risk to cut off when it's less than 75, although looks like that's very unlikely...

Reply:
That makes sense - thanks Tina 👍